ModelicaReference/package.mo

```diff
@@ -4963,6 +5003,37 @@ discontinuously.]</em></p>
 </html>"));
   end 'smooth()';
 
+  class 'spatialDistribution()' "spatialDistribution()"
+    extends ModelicaReference.Icons.Information;
+    annotation (Documentation(info="<html>
+<p>
+Allows the infinite-dimensional problem below to be solved efficiently with good accuracy.
+</p>
+<h4>Syntax</h4>
+<blockquote><pre>
+(out0, out1) = <strong>spatialDistribution</strong>(in0, in1, x, positiveVelocity,
+                               initialPoints = {0.0, 1.0},
+                               initialValues = {0.0, 0.0}); 
+</pre></blockquote>
+<h4>Description</h4>
+<p>
+Many applications involve the modelling of variable-speed transport of properties. 
+One option to model this infinite-dimensional system is to approximate it by an ODE, 
+but this requires a large number of state variables and might introduce either 
+numerical diffusion or numerical oscillations. Another option is to use a built-in 
+operator that keeps track of the spatial distribution of <code>z⁢(x,t)</code>,
+by suitable sampling, interpolation, and shifting of the stored distribution. 
+In this case, the internal state of the operator is hidden from the ODE solver.
+</p>
+
+<p>
+See <a href=\"https://specification.modelica.org/maint/3.4/Ch3.html#spatialdistribution\">
```

Comment:
Since Modelica 3.5 has been released I think it would make more sense to link to that.

Reply:
The problem is that (currently) Modelica Reference is shipped as part of MSL. And as long as MSL is not based on 3.5 I rather stick to 3.4. It might not have an impact in this case but seems more logical to me.